actions/notification_actions.jsx

```diff
@@ -141,19 +141,20 @@ export function sendDesktopNotification(post, msgProps) {
         const activeChannel = getCurrentChannel(state);
         const channelId = channel ? channel.id : null;
         const notify = (activeChannel && activeChannel.id !== channelId) || !state.views.browser.focused;
+        const soundName = user.notify_props !== undefined && user.notify_props.desktop_notification_sound !== undefined ? user.notify_props.desktop_notification_sound : 'Bing';
```

Comment:
as mentioned in a separate comment, I'm not sure about this default if users decides not to use a custom sound.

Reply:
Ok I will keep the old behaviour ;)